open-banking-accelerator/internal-apis/internal-webapps/com.wso2.openbanking.accelerator.event.notifications.endpoint/src/main/java/com/wso2/openbanking/accelerator/event/notifications/endpoint/util/EventNotificationUtils.java

```diff
@@ -97,25 +97,17 @@ public static Response mapEventCreationServiceResponse(EventCreationResponse eve
     public static Response mapEventPollingServiceResponse(EventPollingResponse eventPollingResponse) {
 
         if (EventNotificationConstants.OK.equals(eventPollingResponse.getStatus())) {
-
             return Response.status(Response.Status.OK).entity(eventPollingResponse.getResponseBody()).build();
-
-        } else if (EventNotificationConstants.NOT_FOUND.equals(eventPollingResponse.getStatus())) {
-
-            return Response.status(Response.Status.NOT_FOUND).entity(EventNotificationUtils.getErrorDTO(
-                    EventNotificationEndPointConstants.NOTIFICATIONS_NOT_FOUND,
-                    EventNotificationEndPointConstants.NOT_FOUND_RESPONSE)).build();
-
-        } else if (EventNotificationConstants.BAD_REQUEST.equals(eventPollingResponse.getStatus())) {
-
-            return Response.status(Response.Status.BAD_REQUEST).entity(EventNotificationUtils.getErrorDTO(
-                    EventNotificationEndPointConstants.INVALID_REQUEST,
-                    eventPollingResponse.getErrorResponse())).build();
+        } else {
```

Comment:
we can simplify this if statement to have else if and else right?

Reply:
Else part is used to handle the error scenarios, and there are two cases in error scenarios. hence used if else inside the else part

Reply:
Cool